### ISSUE TITLE
Add a simple instruction how to download or install xi-editor

### ISSUE THREAD

**Issue by rubo77** [google/xi-editor#816]
It is not obvious fo a new visitor, how to install or download a pre compiled package for Windows, Linux or OSX

**Comment by cmyr** [comment]
This is a fair point. We don't currently offer a pre-compiled binary because the editor is not very polished, and we do not want to provide the impression that it is ready to daily use. Making people jump through the hoops of getting it built serves as a minor but maybe useful barrier to entry. :)

**Comment by rubo77** [comment]
so we leave this open until tehere is the first polished version?

would be handy to be informed this way :)

**Comment by cmyr** [comment]
@rubo77 that's a good question. There are various previous issues asking the same question (#7, e.g) , and in any case this is sort of a matter for the various front-ends, so your best bet would be to 'watch' the repo fro the frontend you're interested in. (the frontends are listed in the readme)

So I think I'd be inclined to close this issue?

**Comment by lolgear** [comment]
@cmyr Please, do not close this issue. Better solution for this is header "This project in early stage. We do not offer simple installation instructions right now. Be patient and have fun with Rust :)" in Readme.

I think even first line would be great with Header1 style. ( # )

**Comment by jansol** [comment]
Does #872 address this sufficiently?

**Comment by lolgear** [comment]
@jansol Yes and no. It doesn't contain "This project in early stage. We do not offer simple installation instructions right now. Be patient and have fun with Rust".
These words are necessary in the top of Readme.
Maybe in top of Readme of both core and front-end repositories.

**Comment by matkoniecz** [comment]
Yes, it would be nice to have explicit "It is in early stage and this editor is not ready for daily use as the main editor".

Readme has many hints leading in this direction that I now see, but making it clear would be a good idea.

**Comment by seagle0128** [comment]
Can you please provide the beta pre-compiled binary for the users?

**Comment by nangtrongvuon** [comment]
@seagle0128 The idea is that we don't support releases yet, so we also don't provide a pre-compiled binary. Although it doesn't seem too difficult to just have CI spit out a release build here and there? cc @cmyr ?

**Comment by cmyr** [comment]
If someone wanted to manually package a binary and add it as a github release that would be fine by me.
